src/transformers/models/superpoint/modeling_superpoint.py

```diff
@@ -192,7 +192,7 @@ def forward(
         )
 
 
-class SuperPointInterestPointDecoder(nn.Module):
+class SuperPointKeypointDecoder(nn.Module):
```

Comment:
In case we rename it, it should be renamed across all the code/docs, but it's probably better to leave it with the original name, because it's out of scope for this PR (unless there is any strong reason)

Reply:
Not a necessary change, just by reading it again I thought it would be consistent to rename to this, but I'll open another PR for that matter, no problem